### ISSUE TITLE
Hi!

### ISSUE THREAD

**Issue by trapcodeio** [ezra-obiwale/vuravel-validation#1]
Good Day @ezra-obiwale 😁

I see you are trying to create laravel's kind of validator for vue.
I taught it may interest you to check out a package I wrote similar to laravel validation syntax for javascript.
if it serves your purpose, it may save you some time in composing rules structure.

Package: [Abolish Javascript Validator](http://npmjs.com/package/abolish)
Docs: [Abolish Documentation](https://abolish.trapcode.io)
Playground (Built with vue): [Abolish Playground](https://abolish-playground.trapcode.io)

###### Looks like.
```js
Abolish.validate({
 name: "required|minLenght:2"
})
```

**Comment by ezra-obiwale** [comment]
Hey @trapcodeio 

Thanks for reaching out. I've give it a cursory look and the package looks very good and properly thought out. Good job. 👍️

Wish I'd seen this before starting out though. At any rate, I'll take more look into it and see how I can make it work for me.

Thanks for building and sharing.

**Comment by ezra-obiwale** [comment]
@trapcodeio 

I have gone through the documentation and as I observed earlier, it's a fantastic package. It really would have been great if I had found this before I started writing this package.

Unfortunately, I have gone too far into this package to stop now. I've already created all the in-built rules and the framework to use them. I'm only tidying up things before releasing the first version.

Once again, I appreciate you reaching out and the great work you've done with you package.

Cheers 🙏🏾
